elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -371,11 +371,14 @@
     padding: 32px 64px;
     // Fits as many columns as it can given the constraints
     // @todo evaluate layout needs and if this is a sane default
-    grid-template-columns: 
-      repeat(
-        auto-fill,
-        minmax(#{200 / 16 * 1em}, #{350 / 16 * 1em})
-      );
+    // grid-template-columns: 
+    //   repeat(
+    //     auto-fill,
+    //     minmax(#{200 / 16 * 1em}, #{350 / 16 * 1em})
+    //   );
+    // @todo: still need to test and see if this is a good setting for the flexibility needs of pfe-nav mega menu tray
+    // I tested this setting with 12 sections in the mega menu tray (on Firefox and Chrome) and it worked well. (KS)
+    grid-template-columns: repeat(4, 1fr); // changed to this to fix issues on 1200px brkpt, seems to look better and fit better
```

Comment:
1fr can cause some ... interesting layouts if we get some content that wants to be wide. I'd prefer to avoid it if we can.

Reply:
@wesruv thanks for letting me know! would you mind going over those issues with 1fr with me sometime so that I can understand them and avoid them in the future.